modules/javafx.graphics/src/main/native-glass/mac/GlassLayer.h

```diff
@@ -36,7 +36,8 @@
     andClientContext:(NSObject*)clCtx
          mtlQueuePtr:(long)mtlCommandQueuePtr
       withHiDPIAware:(BOOL)HiDPIAware
-        withIsSwPipe:(BOOL)isSwPipe;
+        withIsSwPipe:(BOOL)isSwPipe
+       useMTLForBlit:(BOOL)useMTLInGlass;
```

Comment:
Minor question on the naming: Since this is only ever set to true when using the SW pipeline, do you think having "Sw" somewhere in the name would be helpful? Given the name, it could be confusing to notice that it is set to false when using the MTL pipeline.